### PR TITLE
Fix link text

### PR DESCRIPTION
**Pull request by rtgoodwin**
~~The build is auto de-referencing login.rackspace.com to "MyRackspace"~~
I'm a goof and hadn't pulled in new `master`. Done and updated PR.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rtgoodwin** [comment]
Hmm. ignore this for the moment. Not clear how MyRack vs Cloud docs are getting displayed.

**Comment by rtgoodwin** [comment]
I hadn't pulled new master into my branch. Good to go now for review/merge.

**Comment by cyrichardson** [comment]
I think maybe it should be MyRackspace.

**Comment by cyrichardson** [comment]
@rtgoodwin  We don’t list Rackspace Portal here: https://developer.rackspace.com/docs/style-guide/control-panels-and-portals/

I don’t see it on the doc com site here: https://www.rackspace.com/login?_ga=2.90667674.189876583.1554128913-4856664.1543960490

File: doc/gettingstarted/add-idp-cp.rst [hunk 1]
```diff
@@ -6,14 +6,14 @@ Add an Identity Provider in the Control Panel
 
 Use the following steps to add an |idp|:
 
-1. Log in to the `MyRackspace Portal <https://login.rackspace.com>`_.
+1. Log in to the `Rackspace Portal <https://login.rackspace.com>`_.
```
Comment (cyrichardson) [inline doc/gettingstarted/add-idp-cp.rst]: I think MyRackspace is correct or maybe it should be Rackspace Customer Portal (based on https://www.rackspace.com/login?_ga=2.186479913.189876583.1554128913-4856664.1543960490) .

Comment (cyrichardson) [reply doc/gettingstarted/add-idp-cp.rst]: Please let me know if we need to update the Style Guide, @rtgoodwin.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-c4012d2915/docs/rackspace-federation/)

**Pull request merged**